public/thank-you.html

```diff
@@ -0,0 +1,117 @@
+    _gaq.push(['_setAccount', 'UA-35433268-5']);
+    _gaq.push(['_trackPageview']);
+
+    (function() {
+     var ga = document.createElement('script'); ga.type = 'text/javascript';
+      ga.async = true;
+     ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+     var s = document.getElementsByTagName('script')[0];
+      s.parentNode.insertBefore(ga, s);
+    })();
+  </script>
+  <link href="/index.css" rel="stylesheet">
+</head>
+
+<body>
+  <div id="signupheader">
+
+    <div id="wrapper">
+      <header id="masthead">
+        <div class="container">
+          <a href="https://www.mozilla.org/" id="tabzilla">mozilla</a> <a href="https://www.mozilla.org" id="header-logo"><img alt="Mozilla Wordmark" class="mozilla-wordmark" src="/img/mozilla-wordmark.png"></a>
+        </div>
+      </header>
+
+      <main>
+        <div class="container">
+          <div id="header">
+            <h1>Thank You 1 placeholder</h1>
+
+            <h2>Thank You 2 placeholder</h2>
+          </div>
+          <div id="white-box" style="display: none;">
+            <h3>Signup confirmation placeholder</h3>
+          </div>
+        </div>
+      </main>
+
+      <footer>
+        <div class="container">
+          <div id="wordmark-and-note">
+            <a href="https://www.mozilla.org"><img alt="Mozilla Wordmark" class="mozilla-wordmark" src="/img/mozilla-wordmark.png"></a>
+
+            <p class="license">Portions of this content are &copy;1998&ndash;2015 by individual mozilla.org contributors. Content available under a <a href=
+            "https://www.mozilla.org/foundation/licensing/website-content/" target="\&quot;_blank\&quot;">Creative Commons license</a>.</p>
+          </div>
+
+          <ul class="links">
+            <li>
+              <a href="https://www.mozilla.org/mission/" target="_blank">Mission</a>
+            </li>
+
+            <li>
+              <a href="https://www.mozilla.org/about/" target="_blank">About</a>
+            </li>
+
+            <li>
+              <a href="https://www.mozilla.org/contact/spaces/" target="_blank">Contact</a>
+            </li>
+
+            <li>
+              <a href="https://www.mozilla.org/privacy/" target="_blank">Privacy Policy</a>
+            </li>
+
+            <li>
+              <a href="https://www.mozilla.org/about/legal/" target="_blank">Legal Notices</a>
+            </li>
+
+            <li>
+              <a class="red" href=
+              "https://sendto.mozilla.org/page/contribute/givenow-seq?preset=2&amp;ref=Advocacy_Website&amp;utm_campaign=Advocacy_Website&amp;utm_source=advocacy.mozilla.org&amp;utm_medium=referral"
+              target="_blank">Donate</a>
+            </li>
+          </ul>
+        </div>
+      </footer>
+    </div>
+  </div><br>
+
+  <div id="signupfooter">
+    <script>
+      (function(){
+        var queryString = decodeURIComponent(window.location.search);
```

Comment:
What are you using the names for? If they're needed, could you use a real querystring parser?